docs/canvas/canvas-workpad.asciidoc

```diff
@@ -1,49 +1,72 @@
-This is where you can make changes to the element.
-+
-[role="screenshot"]
-image::images/canvas_workpad_edit_style.png[]
+. Open *Canvas*.
+
+. Choose one of the following options:
+
+* To start with a single, blank workpad, click *Create workpad*.
+
+* To create a workpad from a template, select *Templates*, then select a preconfigured template.
+
+* To import an existing workpad, click and drag a workpad JSON file to the *Import workpad JSON file* field.
+
+For advanced workpad examples, add a {kibana-ref}/add-sample-data.html[sample Kibana data set], then select *Canvas* from the *View Data* dropdown list.
 
-. Try changing the color palette. Easy right? Click the Back button in your 
-browser to undo your change.
+For more workpad inspiration, go to the link:elastic.co/blog/[Elastic Blog]. 
 
-. Try dragging the chart around the page. Grab the resize handles to make 
-the chart bigger and smaller.
+[float]
+=== Personalize your workpad
 
-. Now click somewhere off of the element, but not on another element, 
-to deselect it. 
+To personalize your workpad, use the editor on the right.
+
+. Add a *Name* to your workpad, for example, `My Canvas Workpad`.
+
+. In the *Width* and *Height* fields, specify the size of your workpad. 
+
+. Select the layout for your workpad. 
 +
-You will notice the sidebar changing again. You now have 
-access to the page and workpad-level settings. Feel free to change the page 
-background color or rename your workpad.
-
-Speaking of pages, Canvas workpads can have multiple pages.
-
-. Click the arrow to the right of the *Page 1 of 2* text in the toolbar at the bottom
-of the page. You'll see a whole new view of the the eCommerce data.
-. Click the *Page 2 of 2* text to see a live preview of all of the pages in your 
-workpad.
-. Click the (+) button to create a new, blank page.
-. Click the *Add element* button in the top right hand corner of the page.
-. Add an *Area Chart.*
+For example, click *720p* for a traditional presentation layout.
+
+. Click the *Background color* picker, then select the background color for your workpad.
 +
-[role="screenshot"]
-image::images/canvas_workpad_3_page.png[]
+image::images/canvas-background-color-picker.gif[]
+
+. If you already know that you want to apply styles to the entire workpad, use the *Global CCS overrides*.
+
+.. Expand *Global CCS overrides*.
+
+.. Enter your text, then click *Apply stylesheet*.
+
+[float]
+=== Share your workpad
+
+When you are ready to share your workpad, create a PDF, or export your workpad to another author.
+
+[float]
+==== Create a PDF of your workpad
+
+To view your workpad outside of Kibana, generate a PDF. 
+
+. If you are using a basic license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Generate PDF*. 
++
+image::images/canvas-generate-pdf.gif[]
+
+[float]
+==== Share your workpad
```

Comment:
Can this title be "Export your workpad" since the title "Share your workpad" already exists?